deployments/container/Dockerfile

```diff
@@ -35,3 +35,4 @@ LABEL summary="Example DRA resource driver for Kubernetes"
 LABEL description="See summary"
 
 COPY --from=build /artifacts/dra-example-kubeletplugin /usr/bin/dra-example-kubeletplugin
+COPY --from=build /artifacts/dra-example-webhook       /usr/bin/dra-example-webhook
```

Comment:
I know it would be generally more desirable to have separate container images for the kubelet plugin and the webhook, but I wasn't 100% sure that's desired here based on the above `make cmds` invocation building all of the packages under `./cmd`. It certainly does simplify some things in the Helm chart if there's only one image in play.